src/main/java/org/alfresco/opencmis/CMISConnector.java

```diff
@@ -970,7 +981,10 @@ protected void onShutdown(ApplicationEvent event)
      * Asynchronously generates thumbnails for the given node.
      *
      * @param nodeRef NodeRef
+     *
+     * @deprecated The thumbnails code is being moved out of the codebase and replaced by the new async RenditionService2 or other external libraries.
      */
+    @Deprecated
     public void createThumbnails(NodeRef nodeRef, Set<String> thumbnailNames)

```

Comment:
This method is a function that does not required any specific implementation.  We will be re-implementing, as required.

Reply:
This one is tricky; you'd have to look where it's used as to whether we deprecate speculatively.